quadlet/minio-prometheus/minio.container

```diff
@@ -0,0 +1,26 @@
+[Unit]
+Description=A minio server contaienr for local S3 compatible storage
+After=local-fs.target network-online.target prometheus-network.service
+Requires=prometheus-network.service
+
+[Container]
```

Comment:
Consider using `ContainerName`. This way you control the names of the containers and don't fail to reach them if at some point Quadlet changes the pattern of its generated container name.

Reply:
[3e5e894](https://github.com/containers/appstore/pull/13/commits/3e5e89492deb348bff32a66ae6d475905e92813b)